libs/input-mapping/src/Table/Options/InputTableOptions.php

```diff
@@ -116,6 +129,11 @@ public function getColumnNamesFromTypes(): array
     public function getStorageApiExportOptions(InputTableStateList $states): array
     {
         $exportOptions = [];
+        if ($this->getSourceBranchId() !== null) {
+            // practically, sourceBranchId should never be null, but i'm not able to make that statically safe and
+            // passing null causes application error in connection, so here is a useless condition.
+            $exportOptions['sourceBranchId'] = $this->getSourceBranchId();
```

Comment:
getSourceBranchId nemůžu dát non-null protože se nastavuje až dodatečně, po tom co tabulka projde nějakým rewriterem. To by se pak mohlo třeba nějak zrefactorovat na to, že to vynutí, že je to nastavený a nebude to vracet null, znamená to ale úpravu kupy testů.